.github/workflows/mypy-type-check.yml

```diff
@@ -39,5 +39,6 @@ jobs:
           tiatoolbox/__main__.py \
           tiatoolbox/typing.py \
           tiatoolbox/tiatoolbox.py \
-          tiatoolbox/utils/ \
-          tiatoolbox/tools/
+          tiatoolbox/utils/*.py \
```

Comment:
using `*.py` excludes files in sub folders for some reason.

Reply:
Oh Ok. I think we will just specify the directory then, mypy will recursively check the directory